src/components/feedback/index.tsx

```diff
@@ -154,8 +223,26 @@ export const Feedback = ({mode, tripPattern, quayListData}: FeedbackProps) => {
     setSubmitted(false);
   }, [tripPattern]);
 
-  if (!category) return null;
   if (submitted) return <SubmittedComponent />;
+  if (!category) return null;
+
+  if (!displayStats || displayStats.length < 1) return null;
+  const statsForCurrentVersion = displayStats.find(
+    (entry) => entry.version === APP_VERSION,
+  );
+  if (!statsForCurrentVersion) return null;
+  if (statsForCurrentVersion.answered) return null;
+
+  const rendersBeforeFeedbackPrompt = 3;
+  const numberOfTimesToAskForFeedback = 3;
```

Comment:
Do you think these constants should be set outside of this file somewhere?

Reply:
I think lifting them to the top of the file is good enough probably

Reply:
Good idea. 

I have now set number of times to ask to 99, after a discussion with Hilde. This number should probably be reduced when Departures v2 no longer is opt-in.